### PR TITLE
OWS styles for Landsat C2 data

### PR DESCRIPTION
**Pull request by eefaye**
Changed files:
 * `style_sr_cfg.py`: 11 styles added for `lsc2_sr`, compiled in a new styles list `styles_lsc2_sr_list`. Scaling has been changed from 0-10000 to 1-65455 and existing ranges scaled accordingly
 * `band_sr_cfg.py`: Common Landsat bands added to a new band list `bands_lsc2_sr` *note it is the same as the `bands_ls` list but so far has been separated so the old Landsat configs can be reorganised on a different occasion*
 * `ows_sr_cfg.py`: "Collection 1" mentions upgraded to "Collection 2". "Five countries" replaced with "all Africa". Style and band list references updated to the new ones mentioned above

**To be added:** `ls_st` configs for surface temperature

Submitting the PR now for review. I expect some whitespace is causing issues but would like to check nothing else is.

### CODE REVIEW

**Comment by eefaye** [comment]
@SpacemanPaul if you could review if this is the correct usage of `index_expression` that would be much appreciated!

File: services/ows_refactored/surface_reflectance/style_sr_cfg.py [hunk 1]
```diff
@@ -759,3 +759,185 @@
...
+        "blue": {"green": 1.0},
+    },
+    "scale_range": [1.0, 19636.0],
+}
+
+style_lsc2_sr_ndvi = {
+    "name": "ndvi",
+    "title": "NDVI - Red, NIR",
+    "abstract": "Normalised Difference Vegetation Index - a derived index that correlates well with the existence of vegetation",
+    "index_expression": "((nir-55000)-(red-55000))/((nir-55000)+(red-55000))",
```
Comment (fangfy) [inline services/ows_refactored/surface_reflectance/style_sr_cfg.py:797]: Should it be 55000? or 0.2/2.75e-5?

**Review by fangfy — CHANGES_REQUESTED** [review]
Fix whitespace and check offset for index calculation. Otherwise looks good.

**Review by fangfy — APPROVED** [review]
This looks good to me. But I don't know what's the build error about.

**Review by fangfy — APPROVED** [review]

**Pull request merged**